runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js

```diff
@@ -567,13 +567,17 @@ class WorkItemDetailPage {
   clickStartCoding  (){
     return this.startCodingElement.click();
   }
-  startCodingDiv (){
-    return element(by.id("start-coding-link"));
+  startCodingDiv  (){
+    return element(by.css(".form-control-static.start-coding"));
+  }
+  startCodingLabel() {
+    return element(by.id("coding_label"));
   }
 
   /** UI elements for Areas */
   get areaLabel (){
-    return element(by.id('area_label'));
+    // return element(by.id('area_label'));
+    return element(by.css('.detail-area-wrap > #area_label'));
```

Comment:
As our CSS has not reached stability, it might be safer to rely on IDs only.
Something like element(by.css('.detail-area-wrap > #area_label'));  would break if we remove detail-area-wrap